iocore/eventsystem/I_SocketManager.h

```diff
@@ -37,6 +37,14 @@
 #include "I_EventSystem.h"
 #include "I_Thread.h"
 
+#ifndef SOCK_NONBLOCK
+#define SOCK_NONBLOCK O_NONBLOCK
+#endif
+
+#ifndef SOCK_CLOEXEC
+#define SOCK_CLOEXEC O_CLOEXEC
```

Comment:
🐼 

That is specified in POSIX.1-2008. This is why we can't have nice things...